lib/package/menu.coffee

```diff
@@ -51,9 +51,10 @@ module.exports =
 
       {type: 'separator'}
 
-      {label: 'Open Plot Pane', command: 'julia-client:open-plot-pane'}
       {label: 'Open Workspace', command: 'julia-client:open-workspace'}
       {label: 'Open Documentation Browser', command: 'julia-client:open-documentation-browser'}
+      {label: 'Open Plot Pane', command: 'julia-client:open-plot-pane'}
```

Comment:
Any reason you changed the ordering here?

Reply:
This is because in my custom-layout PR, the order of those Juno panes are:
Console -> Terminal -> Workspace -> Documentation Browser -> Plot Pane -> Debugger Pane -> Profiler -> Linter

When deciding the order, I naively refered to these orders: https://github.com/aviatesk/atom-julia-client/blame/master/lib/runtime.coffee#L5

Reply:
Ok, fair enough.